scripts/base/init-bare.zeek

```diff
@@ -5802,14 +5802,6 @@ export {
 		## Documentation for this metric.
 		help_text: string &optional;
 
-		## The label names (also called dimensions) of the metric. When
-		## instantiating or working with concrete metrics, corresponding
-		## label values have to be provided. Examples of a label might
-		## be the protocol a general observation applies to, the
-		## directionality in a traffic flow, or protocol-specific
-		## context like a particular message type.
-		labels: vector of string &default=vector();
```

Comment:
Sorry Tim — what's the benefit of removing the labels here and then passing them into the various APIs separately? Can't we just leave them here?

I was just wondering whether a set would be better — then there's no implication of ordering, and it's easier to check membership.

Reply:
I could go either way. I was mostly just trying to remove the duplication of storing the names, but I agree having both results in cleaner function calls.

Reply:
Assuming this field stays, would it still be filled in via the collect methods along with the new table in the other record types?